### PR TITLE
{numlib}[foss/2023a] ensmallen v2.21.1, mlpack v4.3.0

### PR DESCRIPTION
**Pull request by appolloford**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/e/ensmallen/ensmallen-2.21.1-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'ensmallen'
+version = '2.21.1'
+
+homepage = 'https://ensmallen.org/'
+description = """ensmallen is a high-quality C++ library for non-linear numerical optimization"""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```
Comment (smoors) [inline easybuild/easyconfigs/e/ensmallen/ensmallen-2.21.1-foss-2023a.eb:9]: this software looks better suited for the `gfbf` toolchain, as it does not use MPI.

**Comment by smoors** [comment]
Going in, thanks @appolloford!

**Pull request merged**